### PR TITLE
Markdown everywhere

### PR DESCRIPTION
**Pull request by abravalheri**
This PR applies the strategy discussed in #6 to the files generated by the extension.

### CODE REVIEW

**Review by FlorianWilhelm — APPROVED** [review]
Looks really good.

File: README.md [hunk 1]
```diff
@@ -7,22 +7,65 @@
 # pyscaffoldext-markdown
```
Comment (FlorianWilhelm) [inline README.md:7]: The Badge for the Docs status is missing after `Build Status`.

**Review by FlorianWilhelm — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (abravalheri) [reply README.md:7]: Thanks Florian

:P let me add it!

**Comment by abravalheri** [comment]
Thank you for the review @FlorianWilhelm.

I suppose this also closes #1?

**Pull request merged**